infrastructure/src/main/java/org/corfudb/infrastructure/protocol/BaseServer.java

```diff
@@ -0,0 +1,84 @@
+package org.corfudb.infrastructure.protocol;
```

Comment:
Added the new annotated methods in the existing class. I don't see a strong reason to fork the class.